QRCoder/Builders/Payloads/Implementations/StringPayload.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace QRCoder.Builders.Payloads.Implementations
+{
+    public class StringPayload : PayloadBase
```

Comment:
We can use this class for any payload that immediately serializes to a string value.  At least if we design it like `CreateUrl(...)`.  Of course it would be nearly as easy to have additional classes for each payload type, similar to how it is now, if we want `new QRCodeBuilder.Url(...)` or whatnot.